example/all.yaml

```diff
@@ -139,6 +139,7 @@ linter:
     - provide_deprecation_message
     - public_member_api_docs
     - recursive_getters
+    - require_trailing_comma
```

Comment:
We've (almost) standardized on pluralized names so I think I might suggest => `require_trailing_commas`

Reply:
Done, renamed to `require_trailing_commas`.